File: PHP Excel Examples/CellsApiDemo/Cell/Style/setCellStyle.php
<?php

use Spire\Cloud\Excel\Sdk\Api\CellsApi;
use Spire\Cloud\Excel\Sdk\Configuration;
use Spire\Cloud\Excel\Sdk\Model\Border;
use Spire\Cloud\Excel\Sdk\Model\Color;
use Spire\Cloud\Excel\Sdk\Model\Font;
use Spire\Cloud\Excel\Sdk\Model\Style;


$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new CellsApi($configuration);

$name = "SetCellStyle.xlsx";
$folder = "input";
$storage = null;
$sheetName = "Sheet4";
$cellName = "C8";
$style = new Style();
$font = new Font();
$font->setIsBold(true);
$font->setIsItalic(true);
$font->setUnderline("Single");
$font->setName("Algerian");
$font->setSize(8);
$style->setFont($font);
$style->setBackgroundColor(new Color(array('a' => 255, 'r' => 0, 'g' => 255, 'b' => 0)));
$style->setHorizontalAlignment("Center");
$borderColor1 = new Color(array('a' => 255, 'r' => 255, 'g' => 0, 'b' => 0));
$border1 = new Border(array('line_style' => "Medium", 'color' => $borderColor1, 'border_type' => "EdgeTop"));
$borderColor2 = new Color(array('a' => 255, 'r' => 0, 'g' => 255, 'b' => 0));
$border2 = new Border(array('line_style' => "DashDot", 'color' => $borderColor2, 'border_type' => "EdgeRight"));
$style->setBorderCollection(array($border1, $border2));
$apiInstance->setCellStyle($name, $sheetName, $cellName, $style, $folder, $storage);

File: Python Excel Examples/CellsApiDemo/cell/setCellStyle.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration
from spirecloudexcel.api.cells_api import CellsApi

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.cells_api.CellsApi(configuration)
name = "SetCellStyle_1.xlsx"
folder = "/Cells/Cell/Style/"
storage = ""
sheetName = "Sheet4"
cellName = "C8"
style = spirecloudexcel.models.Style()
font = spirecloudexcel.models.Font()
font.is_bold = True
font.is_italic = True
font.underline = "Single"
font.name = "Algerian"
font.size = 8
style.font = font
style.background_color = spirecloudexcel.models.Color(255, 0, 255, 0)
style.horizontal_alignment = "Center" #Left, Right
borders = []
border1 = spirecloudexcel.models.Border("Medium", spirecloudexcel.models.Color(255, 255, 0, 0), "EdgeTop") #EdgeBottom
border2 = spirecloudexcel.models.Border("DashDot", spirecloudexcel.models.Color(255, 0, 255, 0), "EdgeRight") #EdgeLeft
borders.append(border1)
borders.append(border2)
style.border_collection = borders
api.set_cell_style(name, sheetName, cellName, style, folder=folder, storage=storage)